code-reviews/example2/main.py

```diff
@@ -0,0 +1,16 @@
+from math_operations import add, multiply, subtract
+
```

Comment:
create the file called math_operations.py to use as the library to import your functions from